docs/index.rst

```diff
@@ -42,14 +42,15 @@ resources when not actively computing.
 
 .. code-block:: python
 
-   cluster.adapt(minimum=1, maximum=100)
+   cluster.adapt(minimum=6, maximum=90)
```

Comment:
In the cluster.adapt method, I guess that minimum and maximum are expressed in terms of worker processes, not jobs? Maybe this should be highlighted.